src/doomtype.h

```diff
@@ -23,6 +23,7 @@
 
 #include <stddef.h> // size_t, NULL
 #include <stdint.h> // [FG] intptr_t types
+#include <stdbool.h>
```

Comment:
Please remove `#include <stdbool.h>` from `doomtype.h`, we don't use `bool`. There is a `boolean` type in the Doom codebase that has a different size.